src/internal/storage/fileset/postgres_cache.go

```diff
@@ -0,0 +1,122 @@
+package fileset
+
+import (
+	"context"
+
+	proto "github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
+	"github.com/jmoiron/sqlx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/dbutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/track"
+)
+
+// CreatePostgresCacheV1 creates the table for a cache.
+// DO NOT MODIFY THIS FUNCTION
+// IT HAS BEEN USED IN A RELEASED MIGRATION
+func CreatePostgresCacheV1(ctx context.Context, tx *pachsql.Tx) error {
```

Comment:
I hadn't originally intended the V0 names to correspond with 2.0.x, but it makes sense.

Reply:
I can go with v0 if that is more consistent with what we have up to this point. I have some concerns about the migration process, particularly naming migrations without full semver and the packages managing the different versions rather than snapshotting them elsewhere, but I don't think we need to address that now.

Reply:
I'd say keep it as `V1`.  I think most people will also interpret it that way at first glance.